packages/validation/test/validateDictionary/validateDictionary.spec.ts

```diff
@@ -65,9 +65,9 @@ describe('Dictionary - validateDictionary', () => {
 			expect(result.valid).false;
 			assert(result.valid === false);
 
-			expect(result.info.length, 'Only one schema found invalid.').equal(1);
-			expect(result.info[0]?.reason).equal('UNRECOGNIZED_SCHEMA');
-			expect(result.info[0]?.schemaName, 'Correctly names the unrecognized schema.').equal(wrongSchemaName);
+			expect(result.details.length, 'Only one schema found invalid.').equal(1);
+			expect(result.details[0]?.reason).equal('UNRECOGNIZED_SCHEMA');
```

Comment:
Is this worth adding conditional chaining here for this and all similar lines?

e.g.
```suggestion
			expect(result.details?.[0]?.reason).equal('UNRECOGNIZED_SCHEMA');
```

Reply:
The `assert` on line 66 informs us that we have the invalid test result, so we know we will have the `details` property.